nc-news/src/Components/Home.jsx

```diff
@@ -0,0 +1,3 @@
+export const Home = () => {
+    <></>
+}
```

Comment:
This should be taken out of the commit if it's not contributing anything to this ticket.